src/common/sorted-table/index.js

```diff
@@ -695,6 +719,8 @@ export default class SortedTable extends Component {
       />
     )
 
+    const selectedItems = filter(collection, item => selectedItemsIds.has(item.id))
```

Comment:
Shouldn't it be `this._getItems()` instead of `collection` since you'd have fewer items to go through?